<ui version="4.0" >
 <class>FormatConfig</class>
 <widget class="QWidget" name="FormatConfig" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>383</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QFrame" name="m_frame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Scheme :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QComboBox" name="m_selector" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="m_table" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="selectionMode" >
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text" >
       <string>Identifier</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string/>
      </property>
     </column>
     <column>
      <property name="text" >
       <string/>
      </property>
     </column>
     <column>
      <property name="text" >
       <string/>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>O</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string/>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>W</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string/>
      </property>
     </column>
     <column>
      <property name="text" >
       <string/>
      </property>
     </column>
     <column>
      <property name="text" >
       <string/>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
